essentials/items/magic/pickle_rocket.dsc

```diff
@@ -26,9 +26,12 @@ pickle_rocket_handler:
 
     on player right clicks block with:pickle_rocket:
       - determine passively cancelled
-      - inject pickle_launch
+      - inject pickle_launch if:!<player.is_inside_vehicle>
     on player damaged by fall flagged:behr.essentials.pickle_launched:
       - determine cancelled
+    # Prevention from riptide animation playing while player is mounted
+    on player enters entity:
+      - adjust <player> is_using_riptide:false
```

Comment:
might use a flagged switch instead for while this is active
```suggestion
    on player enters entity flagged:behr.essentials.pickle_launched:
      - adjust <player> is_using_riptide:false
```
below, it looks like i forgot to remove the flag at new lines [`[L#44-L#48]`](https://github.com/Purpl-y/b-network/blob/main/essentials/items/magic/pickle_rocket.dsc#L44-L48)
```diff
    - flag player behr.essentials.pickle_launched
    - adjust <player> is_using_riptide:true
    - wait 5t
    - waituntil <player.is_on_ground> || !<player.is_truthy>
-   - flag player behr.essentials.pickle_launched
+   - flag player behr.essentials.pickle_launched:!
```